backend/projects/migrations/0032_metricspreferences.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.0.3 on 2020-03-31 14:56
+
+from django.conf import settings
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('projects', '0031_auto_20191122_1154'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='MetricsPreferences',
+            fields=[
+                ('uuid', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('metrics', django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=100), size=None)),
```

Comment:
Are you sure that this length is enough ? What is the maximal length with all the metrics selected ?